includes/class-edd-html-elements.php

```diff
@@ -489,14 +489,14 @@ public function category_dropdown( $name = 'edd_categories', $selected = 0 ) {
 	 *
 	 * @since 1.5.2
 	 *
-	 * @param string $name         Name attribute of the dropdown
-	 * @param int    $selected     Year to select automatically
-	 * @param int    $years_before Number of years before the current year the dropdown should start with
-	 * @param int    $years_after  Number of years after the current year the dropdown should finish at
-	 *
+	 * @param string $name         Name attribute of the dropdown.
+	 * @param int    $selected     Year to select automatically.
+	 * @param int    $years_before Number of years before the current year the dropdown should start with.
+	 * @param int    $years_after  Number of years after the current year the dropdown should finish at.
+	 * @param string $id A unique identifier for the field.
 	 * @return string $output Year dropdown
 	 */
-	public function year_dropdown( $name = 'year', $selected = 0, $years_before = 5, $years_after = 0 ) {
+	public function year_dropdown( $name = 'year', $selected = 0, $years_before = 5, $years_after = 0, $id = 'edd-year-dropdown' ) {
```

Comment:
Can the `id` use underscores please, just because the `select` method will change them and it feels easier to set it up as it will be output? Same for the month dropdown.

Also, and no real reason why other than I don't care for "dropdown" as a word in general, what if the default id was just `edd_year` or `edd_year_select`? But that's just me.